articles/tech-resources/custom-bounce-domain.md

```diff
@@ -16,37 +16,36 @@ To configure a bounce domain, choose the subdomain you would like to use (e.g. m
+    POST /api/v1/sending-domains/mail.yourdomain.com/verify
 
     {
       "cname_verify" : true
     }
     ```
 
-1. Optionally set the domain as your account default bounce domain so that it is automatically used as the bounce domain for all messages sent through the account.  Here's an example API call to set mail.example.com as the account default bounce domain:
+3. Optionally set the domain as your account default bounce domain so that it is automatically used as the bounce domain for all messages sent through the account.  Here's an example API call to set mail.example.com as the account default bounce domain:
 
     ```
-    PUT /api/v1/sending-domains/mail.example.com
+    PUT /api/v1/sending-domains/mail.yourdomain.com
 
     {
       "is_default_bounce_domain" : true
     }
     ```
 
-Alternately, you may configure multiple CNAME-verified sending domains and select one as the bounce domain on a per message basis by including it in the [transmission `return_path`](https://developers.sparkpost.com/api/transmissions.html#header-transmission-attributes) field for transmission requests using the REST API, or by specifying it in the SMTP MAIL FROM command for SMTP injections.
+Alternatively, you may configure multiple CNAME-verified sending domains and select one as the bounce domain on a per message basis by including it in the [transmission `return_path`](https://developers.sparkpost.com/api/transmissions.html#header-transmission-attributes) field for transmission requests using the REST API, or by specifying it in the SMTP MAIL FROM command for SMTP injections.
 
 After CNAME-verifying or marking the domain as the account default bounce domain, there is up to a 5 minute delay [for compliance verification](https://www.sparkpost.com/docs/getting-started/requirements-for-sending-domains/) before the domain may be used as a message's bounce domain.
 
-If you prefer to use the SparkPost app, then you can create a single bounce domain that will be used for all of your outgoing mail.
+If you prefer to use the SparkPost UI instead of the API to verify a bounce domain, please use the following instructions. Please note, however, that you cannot specify a default bounce domain via the UI; that must be done via the sending-domains API endpoint.
 
-1. Select Account and then Bounce Domains from the main menu.
-1. Click the New Domain button to add a new bounce domain.
-1. Add bounce domain
-1. Select test
-1. Make sure the domain is verified by the green check mark
+1. Select Account and then Sending Domains from the main menu.
+2. If you have a pre-existing sending domain, you can click the "view settings" link to pull up the sending domain's settings. Otherwise, click the "ADD DOMAIN" orange button in the upper right to being configuring a new domain.
+3. Use the "verify DNS records" orange link next to the CNAME record section to verify the domain as a bounce domain.
+4. Make sure the domain is verified by the green check mark.
```

Comment:
We might want to state explicitly that the CNAME section needs a green check mark _and_ the `Status` field at the top of the domain page needs a green `Verified` message.